core/src/test/java/io/parsingdata/metal/AutoEqualityTest.java

```diff
@@ -152,6 +208,7 @@ public class AutoEqualityTest {
     private static final List<Supplier<Object>> TOKEN_ARRAYS = Arrays.asList(() -> new Token[] { any("a"), any("b")}, () -> new Token[] { any("b"), any("c") }, () -> new Token[] { any("a"), any("b"), any("c") });
```

Comment:
Maybe it's time to replace `Arrays.asList` with `List.of`, or do we want to keep Metal Java 8 compatible?

Reply:
Good idea, but maybe not in this issue.